src/components/EventModal/Transfer.vue

```diff
@@ -189,13 +191,16 @@ export default {
     },
   },
   methods: {
+    ...mapActions(['fetchUserInfoByAddress']),
     handleInputToWallet(value) {
       this.toWallet = value;
     },
     handleInputMemo(value) {
       this.memo = value;
     },
-    handleClickTransfer() {
+    async handleClickTransfer() {
+      const userInfo = await this.fetchUserInfoByAddress(this.toWallet);
+      this.recipientName = userInfo.displayName;
```

Comment:
- Please use the `lazyGetUserInfoByAddress` action for better caching
- I think it is better to use a computed property e.g. `toDisplayName` and get the display name with the getter `getUserInfoByAddress(toWallet)`